Source/Extensions/Blazorise.Components/DropdownList.razor

```diff
@@ -1,18 +1,33 @@
-﻿@typeparam TItem
+﻿@using Microsoft.AspNetCore.Components.Web.Virtualization
+@typeparam TItem
 @typeparam TValue
 <Dropdown @ref="@dropdownRef" ElementId="@ElementId" Class="@Class" Style="@Style" RightAligned="@RightAligned" Disabled="@Disabled" Direction="@Direction" Attributes="@Attributes">
     <DropdownToggle @ref="@dropdownToggleRef" Color="@Color" TabIndex="@TabIndex">@ChildContent</DropdownToggle>
     <DropdownMenu MaxMenuHeight="@MaxMenuHeight">
         @if ( Data != null )
         {
-            foreach ( var item in Data )
+            @if ( Virtualize && Data is ICollection<TItem> collectionableData )
```

Comment:
Do we not have an util for this? Or we very specifically only want the ICollection derivative?

Reply:
I wish, but `Virtualize` accepts `ICollection` type only.